sdk/formrecognizer/Azure.AI.FormRecognizer/samples/README.md

```diff
@@ -19,6 +19,10 @@ Azure Cognitive Services Form Recognizer is a cloud service that uses machine le
 - Custom - Build custom models to extract text, field values, selection marks, and table data from documents. Custom models are trained with your own data, so they're tailored to your documents.
 
 ## Common scenarios samples for SDK
+- [Analyze the layout of a document](https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/formrecognizer/Azure.AI.FormRecognizer/samples/Sample_AnalyzeLayout.md)
+- [Analyze a general document](https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/formrecognizer/Azure.AI.FormRecognizer/samples/Sample_AnalyzeDocument.md)
```

Comment:
We had used "general document" in python, but it didn't seem to explain the feature much to me... so we switched to referring to it as "prebuilt document" to align with the service. IMO "prebuilt" tells me that that I don't need to train a model, whereas "general" doesn't tell me a lot. Regardless of what we go with, I think we should try to align across languages for beta.1. @catalinaperalta @maririos @samvaity @witemple-msft

Reply:
prebuilt document sounds good to me

Reply:
Updated the samples to use the "prebuilt document" term in .NET as well.